grove/blackbox_algorithms/deutsch_jozsa.py

```diff
@@ -0,0 +1,69 @@
+"""Module for Deutsch-Jozsa Algorithm. Adapted from Harley Patton's deutsch_jozsa.py"""
+import pyquil.quil as pq
+from pyquil.gates import *
+import utils as bbu
+from blackbox import AbstractBlackBoxAlgorithm
+
+class DeutschJozsa(AbstractBlackBoxAlgorithm):
```

Comment:
If we are going to merge in @harleypatton  `DeutschJozsa`, we shouldn't code review this twice. If this is the way you all think it should be written, this should be broken off into its own PR dependent on `AbstractBlackBoxAlgorithm`.